server/apps/immich/src/api-v1/asset/response-dto/map-marker-response.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { AssetEntity } from '@app/infra/entities';
+import { roundToDecimals } from '../../../utils/coordinate.util';
+
+export type MapMarkerResponseDto = [
+  // latitude
+  number,
+  // longitude
+  number,
+  // assetId
+  string?,
+];
+
+export function mapAssetMapMarker(asset: AssetEntity, preload: boolean): MapMarkerResponseDto {
+  const lat = asset.exifInfo?.latitude || 0;
+  const lon = asset.exifInfo?.longitude || 0;
+
+  const response = [roundToDecimals(lat, 5), roundToDecimals(lon, 5)] as MapMarkerResponseDto;
```

Comment:
This should be a regular type annotation instead of a cast. And for rounding we can also use Lodash, instead of having to create our own implementation